__tests__/pages/login.test.tsx

```diff
@@ -0,0 +1,50 @@
+import LoginPage from '../../src/pages/login';
+import { render, fireEvent } from '@testing-library/react';
+
+describe('LoginPage', () => {
+    it('should render without throwing an error', () => {
+        const { container } = render(<LoginPage />);
+        expect(container).toMatchSnapshot();
+    });
+
+    it('should render username input', () => {
+        const { getByLabelText } = render(<LoginPage />);
+        const usernameInput = getByLabelText('Username') as HTMLInputElement;
+        expect(usernameInput).toBeInTheDocument();
+        expect(usernameInput).toHaveAttribute('type', 'text');
+    });
+
+    it('should render password input', () => {
+        const { getByLabelText } = render(<LoginPage />);
+        const passwordInput = getByLabelText('Password') as HTMLInputElement;
+        expect(passwordInput).toBeInTheDocument();
+        expect(passwordInput).toHaveAttribute('type', 'password');
+    });
```

Comment:
can we not combine these two test case into one and test it

Reply:
No

Reply:
why sir?